config/config.go

```diff
@@ -31,8 +31,8 @@ type CMD struct {
 }
 
 type SSH struct {
-	Environments   []Environment `yaml:"environments"`
-	User           string        `yaml:"user"`
+	Environments []Environment `yaml:"environments"`
```

Comment:
I wonder if we should move the environments out of the SSH struct now that the list is used for more than just SSH things?